src/Driver.hpp

```diff
@@ -358,6 +407,13 @@ class Driver
      */
     int readPacket(uint8_t* buffer, int bufsize, int packet_timeout, int first_byte_timeout = -1);
 
+    /** @overload @deprecated
```

Comment:
Because I copy/pasted the documentation from the readPacket above (which *is* deprecated). I'm going to refactor that stuff to overload the non-deprecated readPacket that uses base::Time for timeouts) and update.